src/util/templates/EjsTemplateEngine.ts

```diff
@@ -11,21 +11,27 @@ import Dict = NodeJS.Dict;
  */
 export class EjsTemplateEngine<T extends Dict<any> = Dict<any>> implements TemplateEngine<T> {
   private readonly applyTemplate: Promise<TemplateFunction>;
+  private readonly rootPath: string;
 
   /**
    * @param template - The default template @range {json}
    */
-  public constructor(template?: Template) {
+  public constructor(baseUrl: string, template?: Template) {
     // EJS requires the `filename` parameter to be able to include partial templates
     const filename = getTemplateFilePath(template);
+    this.rootPath = new URL(baseUrl).pathname;
+
     this.applyTemplate = readTemplate(template)
       .then((templateString: string): TemplateFunction => compile(templateString, { filename }));
   }
 
   public async render(contents: T): Promise<string>;
   public async render<TCustom = T>(contents: TCustom, template: Template): Promise<string>;
   public async render<TCustom = T>(contents: TCustom, template?: Template): Promise<string> {
-    const options = { ...contents, filename: getTemplateFilePath(template) };
-    return template ? render(await readTemplate(template), options) : (await this.applyTemplate)(options);
+    const options = { ...contents, filename: getTemplateFilePath(template), rootPath: this.rootPath };
+    return template ?
+      render(await readTemplate(template),
+        options) :
+      (await this.applyTemplate)(options);
```

Comment:
Your IDE has some interesting opinions about formatting if it changes the line to this 😉

Reply:
This was auto-fixed through ESLint extension I believe :P struggling between the max allowed width of a line and the indentation/newline restrictions in statements. Should be better now in latest changes though